### PR TITLE
[CheshireEast] allow cleaning reports to be made anywhere

### PR DESCRIPTION
**Pull request by struan**
Fixes https://github.com/mysociety/fixmystreet-commercial/issues/2050

[skip changelog]

### CODE REVIEW

File: web/cobrands/cheshireeast/assets.js [hunk 1]
```diff
@@ -62,7 +76,15 @@ fixmystreet.assets.add(defaults, {
     all_categories: true,
     actions: {
         found: fixmystreet.message_controller.road_found,
-        not_found: fixmystreet.message_controller.road_not_found
+        not_found: function(layer) {
+            var cat = $('select#form_category').val();
+          console.log(cat, cleaning_cats);
```
Comment (dracos) [inline web/cobrands/cheshireeast/assets.js:81]: debug

File: web/cobrands/cheshireeast/assets.js [hunk 2]
```diff
@@ -62,7 +76,15 @@ fixmystreet.assets.add(defaults, {
...
+            if (OpenLayers.Util.indexOf(cleaning_cats, cat) != -1) {
+              fixmystreet.message_controller.road_found();
```
Comment (dracos) [inline web/cobrands/cheshireeast/assets.js:83]: Should perhaps pass layer in (though I realise not used)?

**Review by dracos — CHANGES_REQUESTED** [review]
Looks fine. I guess one question (presumably considered) is that if the highways layer now doesn't apply to all categories, why not list the categories it does apply to and stop it being `all_categories`, instead of providing an exemption list? Is that list much longer?

**Comment by struan** [comment]
The list of categories it applies to is much longer so this seemed the sensible way round.

**Review by dracos — APPROVED** [review]

**Pull request merged**